spec/spec_helper_acceptance.rb

```diff
@@ -164,7 +163,6 @@ def teardown_puppet_on(host)
 package { ['puppet-agent', 'puppet', 'mcollective', 'mcollective-client']: ensure => purged }
   EOS
   on host, puppet('apply', '-e', "\"#{pp}\"")
-  remove_defaults_on host
```

Comment:
I should debug more, but it just caused everything to fail when the config file set `type: aio`.